testsuite/MDAnalysisTests/core/test_atomgroup.py

```diff
@@ -56,6 +57,7 @@ def test_AtomGroupUniverse_usage_warning(self):
         with pytest.deprecated_call():
             mda.core.AtomGroup.Universe(PSF, DCD)
 
+
```

Comment:
Can you add tests for the changed behavior (returning `self`)
- `AtomGroup.atoms`
- `ResidueGroup.residues`
- `SegmentGroup.segments`

along the lines of
```python
assert ag.atoms is ag
```

Reply:
It's already covered in `test_groups.py` in the class `TestGroupLevelTransition`, see my other comments [here](https://github.com/MDAnalysis/mdanalysis/pull/1922/files#r195831444), [here](https://github.com/MDAnalysis/mdanalysis/pull/1922/files#r195831780), and [here](https://github.com/MDAnalysis/mdanalysis/pull/1922/files#r195831909).